BUILD.md

```diff
@@ -5,21 +5,19 @@ This project consists of two components:
 1. PyMACER. PyMACER is a Python adaptation of [MACER: A Modular Framework for Accelerated Compilation Error Repair.](https://github.com/purushottamkar/macer).
 2. vscode-extension. This provides the frontend experience in Visual Studio Code, and uses PyMACER.
 
-Both of these components are found in the `ai-compile` directory.
+Both of these components are in the `ai-compile` directory.
 
 ## Pre-requisities
 
 - Python 3.8.9 (64-bit). Python 3.9 does not work due to tensorflow requirement.
 - Visual Studio 2019 with Desktop development with C++.
-- Some recent version of node and npm for building the vscode-extension.
+- Recent version of node and npm for building the vscode-extension.
```

Comment:
Can we specify exact versions that are compatible? "Some recent version" is vague and confusing.

Reply:
I have no idea which exact versions are compatible.

Reply:
That's what I assumed and hence just removed "Some".